### DISCUSSION TITLE
How to use topsApp.py to handle only a small area?

### DISCUSSION THREAD

**Discussion by Nirdesh31** [isce-framework/isce2#811] [General]
Hello,
I am trying to run topsApp.py but its taking a lot of time executing as it is processing the whole file. Is there any way that i could process only certain region of the image ?

**Comment by Karthi-Wolf** [comment]
i think you can. while using stackSentinel.py we can specify the boundary box using "-b" like following
"stackSentinel.py -s /SLCs -o /orbits -d demLat_N19_N22_Lon_E083_E086.dem.wgs84 -b '20.5 21.0 85.5 86.0' -W slc -n 1 -a /orbits -C geometry -e 0.65 -m 20240822 --num_proc 6"

**Comment by Ayxun** [comment]
Hi, I have a question. The merged/filt_topophase.unw.geo generated by topsApp is processed by the following two commands:
gdal_translate -of GMT -b 2 filt_topophase.unw.geo filt_topophase.unw.geo.grd
gmt grdmath filt_topophase.unw.geo. grd-0.44 MUL = los.grd
So is los.grd in cm or mm？
Thank you for your clarification
